docs/custom-music/index.md

```diff
@@ -113,22 +114,28 @@ Once done, save and repackage your mod, and you'll find your audio replacement p
 
 ## List of Wwise IDs for the soundtrack
 ### No Rest For The Wicked / BranchBank
-| Name                                                                    | Wwise Id   | Notes |
-|:------------------------------------------------------------------------|:-----------|:------|
-| BranchBank_Anticipation_Intro_130bpm_16bars                             | 653308651  |       |
-| BranchBank_Anticipation_Loop_130bpm_2bars                               | 382346082  |       |
-| BranchBank_Assault_Intro_130bpm_2bars                                   | 48507945   |       |
-| BranchBank_Assault_Loop_v02                                             | 629662482  |       |
-| BranchBank_Control_Loop_130bpm_32bars                                   | 653896124  |       |
-| BranchBank_Transition_Assault_to_Control_130bpm_2Bar_2beatsPreEntry_v02 | 632657818  |       |
-| BranchBank_Transition_Assault_to_Control_02_130bpm_1beatPreEntry        | 786633665  |       |
-| BranchBank_Transition_Assault_C_to_Control_130bpm_1beatPreEntry_v03     | 160788638  |       |
-| BranchBank_Transition_Assault_D_to_Control_130bpm_1beatPreEntry         | 377552255  |       |
-| BranchBank_Transition_Stealth01_to_Control_130bpm_2barsPreEntry         | 725612459  |       |
-| BranchBank_Transition_Stealth01_to_Stealth02_130bpm_2beatsPreEntry      | 839959304  |       |
-| BranchBank_Transition_Stealth02_to_Stealth03_130bpm_3beatsPreEntry      | 793440025  |       |
-| BranchBank_Transition_Stealth03_to_Stealth04_130bpm_1Bar_2beatsPreEntry | 1057654261 |       |
-| BranchBank_Transition_Stealth04_to_Stealth05_130bpm_1Bar_2beatsPreEntry | 532610310  |       |
+| Name                                                                    | Wwise Id   | Notes                                                              |
+|:------------------------------------------------------------------------|:-----------|:-------------------------------------------------------------------|
+| BranchBank_Anticipation_Intro_130bpm_16bars_v02                         | 856428028  | 1nd half 0s-20s                                                    |
+| BranchBank_Anticipation_Intro_130bpm_16bars                             | 653308651  | 2nd half 25.9s-29.5s                                               |
+| BranchBank_Anticipation_Loop_130bpm_2bars                               | 382346082  |                                                                    |
+| BranchBank_Assault_Intro_130bpm_2bars                                   | 48507945   |                                                                    |
+| BranchBank_Assault_Loop_v02                                             | 629662482  | Sequence 0.0s-0.1s-59.170s, 1m58.260s-3m11.850s, 59.180s-1m58.250s |
```

Comment:
The note here is confusing. Is that when exactly it plays in game, or is the loop cut up like that?